### DISCUSSION TITLE
Internationalisation question

### DISCUSSION THREAD

**Discussion by hvitis** [PxlSyl/tailwind-nextjs-starter-blog-i18n#4] [General]
**Is your feature request related to a problem? Please describe.**
Hey! I have seen that there is a fallback language. Does it mean that this repo is for a bilingual blog or multilingual?

**Comment by PxlSyl** [comment]
Hello! It is programmed as a template with a base of two languages, but it can easily be extended to other languages (3, 4, 5 or even more) by modifying a few files. (I made sure that it wasn't that complicated to set up) 

I plan to explain this in a future article which will be on the demo :)

That said, I think that two languages will cover most cases of uses: English as a conventional language and technical language, and the natural language of the blogger. More could be a bit overkill for a blog lol, but if you need three, it's totally possible!

The fallback language is actually designed as the language without a locale in the url, like this : "yoursite/" instead of "yoursite/en" and is set up to english. (implementation in middleware.ts file) But it can be set up to any language.

Let me know if you need more infos!

**Reply by hvitis** [reply]
Article with a demo would be awesome. Definitely do it!

The fallback language is genious. Chapeaux bas.

**Comment by hvitis** [comment]
Hey! I see now. It's super useful. Many thanks for that. Do you need help in PRs or maintaining? I'm planning to make it multilingual for my purposes.

**Comment by PxlSyl** [comment]
Well, if you have any new ideas, or see any bugs, or things that should be improved, that's totally welcome!

**Reply by hvitis** [reply]
I think that your idea of adding to this template is so amazing and the implementation so good that it needs to have a better readme for newbies. I remember the time I put into struggling with internationalisation and here it is. Any coffee I can buy you or smth? 
When it comes to other things the multiple languages with easy support would be nice. I can do PR

**Reply by PxlSyl** [reply]
@hvitis Do you still want a blog in 3 or 4 languages? I can do it for you and deliver you a private template, you can contact me through my professional site, if you want to discuss it! (Don't worry, it won't be very expensive) :)

**Comment by PxlSyl** [comment]
Yes it's quite painful loool I spent a lot of time with this too! But don't worry, the project being quite recent I didn't have time to create a better read.me, so I just wrote something basic...

It is already planned, first with an article on the demo (English and French) and then with the same articles in the repository.

I haven't set up github sponsorship yet, I added my ko-fi link just in case (something I already wanted to do but had put aside)

In any case, I'm happy that this project can benefit other people, and your enthusiasm is very infectious lol

Okay for PR if you see some things that needs improvments.

Edit : I just finished writing the new documentation. If some things are not clear enough, don't hesitate! (And if you need help for your own multilanguage implementation, feel free to ask)
